### DISCUSSION TITLE
Review of Macros

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#235] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/macros.svg">

**Comment by eliotttak** [comment]
I can't get my script to run automatically in a loop.
I also tried putting a `setInterval()` function (not `while(true)`) so as not to bug the browser), but it didn't work.

**Reply by Misokados** [reply]
Hello,
please provide us with a test file and describe the usage scenario step-by-strep, we will check it out.
